projects/storefrontapp-e2e-cypress/cypress/integration/vendor/cdc/profile-update/profile-update.e2e-spec.ts

```diff
@@ -0,0 +1,23 @@
+import * as cdc from '../../../../helpers/vendor/cdc/cdc';
+
+describe('Register', () => {
+  Cypress.on('uncaught:exception', (err, runnable) => {
+    return false;
+  });
+  before(() => {
+    cy.window().then((win) => win.sessionStorage.clear());
+    cy.visit('/cdc/login');
+  });
+
+  it('should update and redirect to login page', () => {
+    cdc.registerUser();
+    cdc.verifyRegistrationSuccess();
```

Comment:
Can we move these 2 to `before` part?

Reply:
I have refactored it to re-use the same SampleUser details across the tests. First we register the customer, then use the same customer to login, and finally login and update the profile.